### DISCUSSION TITLE
Hardware Recommendations

### DISCUSSION THREAD

**Discussion by simondrake** [blakeblackshear/frigate#4422] [Ask A Question] [answered]
I'll prefix my question by saying I **have** looked at the [Recommended Hardware](https://docs.frigate.video/hardware) section of the Frigate docs, but most (if not all) of the options aren't available in the UK.

Currently, I'm running Frigate as a HassOS add-on, which itself is running in a ProxMox VM. I'm aware this isn't recommended, but it was a quick-setup to test everything out.

Now I've used it a bit, and can see how awesome Frigate is, I'm looking to set-up a new bare-metal server, running Ubuntu, where I'll install Frigate/HomeAssistant in Docker containers, probably utilising a NFS share on a NAS to store the snapshots/recordings.

For a while, at least, Frigate will be running without Google Coral, because they are out of stock or way above MSRP. 

I'm thinking of getting a cheap little Dell Optiplex (like [this](https://www.ebay.co.uk/itm/185656450247?hash=item2b39fc9cc7%3Ag%3A3WMAAOSw5FBjcTLW&amdata=enc%3AAQAHAAAAoDQFekqpfDj4tCKU2omHjQ5gjGPkkfdkpxa4WdwV2M0JV%2FyIGaHb7IqpvycXvuJX8PtlLp94lYkqoU9Z4AHV6ORVkfeiVH6zop2Uw1CYYGuMkBay%2Fa8O4Cb8Fo2%2BCdNW6wtpiaZcZrwLWo74gGoNynOJA%2Fu%2FoGhZ2frxiEV6l%2FoXR3jToXcKBUYMFUZarVIctkEsICxLv%2BPcqVvLaBGk%2FGw%3D%7Ctkp%3ABk9SR5a0u92QYQ&LH_Auction=1) one), but thought it was worth checking in to see if there would be better alternatives.

From what I can see it doesn't need a huge box, and I'll only be running Frigate/HA, so I wasn't looking to break the bank.

**Comment by NickM-27** [comment, marked as answer]
Looks like it does support quick sync video so you should be good with that: https://ark.intel.com/content/www/us/en/ark/products/97485/intel-core-i37100t-processor-3m-cache-3-40-ghz.html

as far as cameras go, even if the exact cameras aren't available the advice on having a camera that is customizable with multiple substreams is still the main advice

**Reply by simondrake** [reply]
> Looks like it does support quick sync video so you should be good with that: https://ark.intel.com/content/www/us/en/ark/products/97485/intel-core-i37100t-processor-3m-cache-3-40-ghz.html

Is that the general advice for a sever? Get one with a chip that supports Quick Sync? I'm assuming it won't need a large amount of memory as it's mostly CPU work.

> as far as cameras go, even if the exact cameras aren't available the advice on having a camera that is customizable with multiple substreams is still the main advice

Thanks :) I've actually got the cameras.

**Reply by NickM-27** [reply]
> Is that the general advice for a sever? Get one with a chip that supports Quick Sync? I'm assuming it won't need a large amount of memory as it's mostly CPU work.

Depends what you define as a large amount of memory and how many cameras you have, but yes frigate requires decoding the stream which is a non-negligible amount of work. Having an integrated GPU that supports QSV makes that easier.

**Reply by simondrake** [reply]
> Depends what you define as a large amount of memory and how many cameras you have, but yes frigate requires decoding the stream which is a non-negligible amount of work. Having an integrated GPU that supports QSV makes that easier.

I wouldn't be planning on getting anything with less than 8GB memory. I have four cameras which are actually running pretty well on the HAOSS vm inside ProxMox, it's just not a long-term solution.

IIRC from researching QSV a while ago, when I was setting up a different service, version 6 and above is recommended as before that it doesn't support HEVC so a 7x series or above is better. I assume the same applies to Frigate? 

Thanks for all the replies :)

**Reply by NickM-27** [reply]
yes, I don't really think `future-proofing` is a good reason to buy things in general but in this case definitely recommend one that supports decoding hevc.

**Reply by simondrake** [reply]
Yeah, it's not like a 7x series would be overly expensive or unavailable considering we're on the 13th (??) generation now.

Thanks for all the answers @NickM-27 !

**Reply by simondrake** [reply]
To clarify something I said before (mostly for future readers)

> version 6 and above is recommended as before that it doesn't support HEVC so a 7x series or above is better.

Should have been "a 6x series or above is better". Skylake is considered the minimum recommended QuickSync.
